todoapp/todos/views.py

```
@@ -1,25 +1,69 @@
+
+    def get_queryset(self):
+        user = self.get_user()
+        if user:
+            return todos_models.Todo.objects.filter(user=user).order_by("-date_created")
+        return todos_models.Todo.objects.all()
 
+    def perform_create(self, serializer):
+        user = self.get_user()
+        serializer.save(user=user)
 
+    def destroy(self, request, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `destroy`

(ANN201)